xtask/src/codegen/gen_assists_docs.rs

```diff
@@ -46,9 +46,7 @@ fn reveal_hash_comments(text: &str) -> String {
 
 fn collect_assists() -> Result<Vec<Assist>> {
     let mut res = Vec::new();
-    for entry in fs::read_dir(project_root().join(codegen::ASSISTS_DIR))? {
-        let entry = entry?;
-        let path = entry.path();
+    for path in rust_files(&project_root().join(codegen::ASSISTS_DIR)) {
         if path.is_file() {
```

Comment:
I think we can now remove this if? Otherwise, r=me

bors d+

Reply:
Good catch, I've removed this `if`.